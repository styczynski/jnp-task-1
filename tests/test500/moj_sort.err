Error in ./tests/test500/idFile, line 1: xzxx0000
Error in ./tests/test500/idFile, line 15: xzxx1400
Error in ./tests/test500/idFile, line 18: xzxx1700
Error in ./tests/test500/idFile, line 22: xzxx2100
Error in ./tests/test500/idFile, line 29: xzxx2800
Error in ./tests/test500/idFile, line 30: xzxx2900
Error in ./tests/test500/idFile, line 37: xzxx3600
Error in ./tests/test500/idFile, line 38: xzxx3700
Error in ./tests/test500/idFile, line 39: xzxx3800
Error in ./tests/test500/idFile, line 48: xzxx4700
Error in ./tests/test500/idFile, line 52: xzxx5100
Error in ./tests/test500/idFile, line 53: xzxx5200
Error in ./tests/test500/idFile, line 54: xzxx5300
Error in ./tests/test500/idFile, line 61: xzxx6000
Error in ./tests/test500/idFile, line 67: xzxx6600
Error in ./tests/test500/idFile, line 74: xzxx7300
Error in ./tests/test500/idFile, line 86: xzxx8500
Error in ./tests/test500/idFile, line 89: xzxx8800
Error in ./tests/test500/idFile, line 95: xzxx9400
Error in cin, line 2: grupa9/zadanie3/aa560000+aa770000
Error in cin, line 3: grupa4/zadanie4/xzxx3800+aa160000
Error in cin, line 5: grupa9/zadanie1/aa420000+aa420000+aa420000
Error in cin, line 6: aa710000
Error in cin, line 7: aa400000
Error in cin, line 9: grupa9/zadanie6/xzxx3600+xzxx3600
Error in cin, line 10: grupa6/zadanie6/aa780000+xzxx2900
Error in cin, line 11: aa160000
Error in cin, line 12: grupa7/zadanie4/aa740000+xzxx2900+xzxx2900
Error in cin, line 14: grupa5/zadanie2/xzxx2100+aa410000+xzxx2100
Error in cin, line 15: grupa9/zadanie3/aa350000+aa020000
Error in cin, line 16: grupa1/zadanie5/xzxx1700+aa990000
Error in cin, line 17: aa330000
Error in cin, line 18: aa820000
Error in cin, line 19: aa220000
Error in cin, line 20: grupa5/zadanie4/xzxx0000+aa040000+xzxx0000
Error in cin, line 22: aam-5900
Error in cin, line 22: aam-5900
Error in cin, line 23: grupa8/zadanie6/xzxx3700+aa740000
Error in cin, line 24: aa710000
Error in cin, line 24: aa710000
Error in cin, line 25: grupa1/zadanie6/xzxx1700+xzxx1700+xzxx1700
Error in cin, line 26: gru pa5/zadanie3/aa570000+aa490000+aam-4300
Error in cin, line 27: grupa6/zadanie3/aa560000+xzxx1400+xzxx1400
Error in cin, line 28: grupa2/zadanie2/aa770000+xzxx5100
Error in cin, line 29: grupa2/zadanie1/xzxx6600+xzxx6600+aa690000
Error in cin, line 30: grupa9/zadanie3/aam-8400+aa150000+aa390000
Error in cin, line 31: grupa5/zadanie1/xzxx6600+aam-8400+xzxx0000
Error in cin, line 33: aa970000
Error in cin, line 34: grupa4/zadanie2/xzxx8500+xzxx0000
Error in cin, line 35: aa090000
Error in cin, line 36: grupa3/zadanie4/aa890000+xzxx5100+aa890000
Error in cin, line 37: aa990000
Error in cin, line 38: aa070000
Error in cin, line 39: aa570000
Error in cin, line 39: aa570000
Error in cin, line 40: aa160000
Error in cin, line 42: grupa7/zadanie4/aa300000+xzxx3800+xzxx5100
Error in cin, line 45: grupa1/zadanie1/aam-1100+xzxx9400+aam-1100
Error in cin, line 46: grupa9/zadanie1/aa230000+xzxx8800
Error in cin, line 47: grupa8/zadanie2/xzxx9400+xzxx2800+xzxx9400
Error in cin, line 48: grupa2/zadanie6/xzxx2900+xzxx2900
Error in cin, line 49: grupa8/zadanie5/xzxx8800+xzxx1700
Error in cin, line 50: grupa9/zadanie4/xzxx5200+xzxx5200+xzxx5200
Error in cin, line 51: grupa5/zadanie6/xzxx3600+xzxx3600+aa770000+aam-2600+xzxx3600
Error in cin, line 52: grupa3/zadanie1/aa230000+aa830000+xzxx6600
Error in cin, line 53: aa410000
Error in cin, line 54: aa790000
Error in cin, line 55: aa350000
Error in cin, line 56: aa230000
Error in cin, line 56: aa230000
Error in cin, line 57: aa810000
Error in cin, line 58: aa090000
Error in cin, line 59: grupa9/zadanie3/xzxx1700+aa710000
Error in cin, line 61: aa620000
Error in cin, line 62: aa990000
Error in cin, line 63: aa390000
Error in cin, line 64: grupa3/zadanie4/xzxx9400+xzxx9400+aam-5500
Error in cin, line 65: aa320000
Error in cin, line 66: aa310000
Error in cin, line 67: aa070000
Error in cin, line 68: aa780000
Error in cin, line 69: aa050000
Error in cin, line 71: aam-9300
Error in cin, line 72: grupa9/zadanie3/aa640000+aa980000+aa680000
Error in cin, line 73: grupa9/zadanie1/aa970000+aa310000+aa310000
Error in cin, line 75: grupa3/zadanie5/xzxx4700+xzxx9400
Error in cin, line 76: aam-8400
Error in cin, line 77: aa070000
Error in cin, line 77: aa070000
Error in cin, line 78: grupa9/zadanie2/aam-8400+aam-8400
Error in cin, line 79: aa200000
Error in cin, line 81: aa270000
Error in cin, line 82: grupa1/zadanie3/aam-4300+xzxx8500
Error in cin, line 84: aam-0800
Error in cin, line 84: aam-0800
Error in cin, line 85: grupa5/zadanie5/xzxx7300+xzxx7300
Error in cin, line 86: grupa3/zadanie3/xzxx8800+xzxx4700
Error in cin, line 87: aa580000
Error in cin, line 89: aa950000
Error in cin, line 90: aam-0800
Error in cin, line 92: aa750000
Error in cin, line 93: aam-8400
Error in cin, line 94: grupa6/zadanie5/aa790000+aa030000+xzxx0000
Error in cin, line 95: aa220000
Error in cin, line 96: aa120000
Error in cin, line 97: grupa5/zadanie5/xzxx2800+aa720000
Error in cin, line 98: grupa7/zadanie2/xzxx5200+xzxx4700+xzxx4700
Error in cin, line 99: grupa9/zadanie4/aa750000+aam-1800+aa580000
Error in cin, line 100: aa490000
Error in cin, line 102: grupa4/zadanie1/xzxx3700+aam-1800+xzxx3700
Error in cin, line 103: grupa5/zadanie4/aa050000+xzxx2100
Error in cin, line 105: aa100000
Error in cin, line 106: aa990000
Error in cin, line 107: grupa2/zadanie1/aa750000+xzxx2800+xzxx2800
Error in cin, line 111: aa010000
Error in cin, line 112: grupa4/zadanie5/xzxx8800+aam-4500
Error in cin, line 113: aa950000
Error in cin, line 114: grupa3/zadanie5/xzxx5300+aa870000+xzxx5300
Error in cin, line 115: grupa7/zadanie6/aa500000+aa160000+aa160000+xzxx5300
Error in cin, line 116: aam-9000
Error in cin, line 117: aa960000
Error in cin, line 118: grupa7/zadanie1/aa650000+aa650000+aa650000+aa060000+aa060000
Error in cin, line 119: grupa9/zadanie2/aa250000+aa240000+aa250000
Error in cin, line 120: aa580000
Error in cin, line 121: grupa2/zadanie3/xzxx1700+xzxx1700
Error in cin, line 122: grupa9/zadanie3/aa440000+aa440000
Error in cin, line 123: aa230000
Error in cin, line 124: grupa4/zadanie6/xzxx8500+aam-1100
Error in cin, line 125: aa390000
Error in cin, line 126: aa100000
Error in cin, line 129: aa990000
Error in cin, line 130: grupa4/zadanie3/xzxx5200+aa830000+xzxx5200
Error in cin, line 131: aa630000
Error in cin, line 132: grupa9/zadanie6/aa190000+aa490000
Error in cin, line 134: grupa2/zadanie1/aam-1100+xzxx9400
Error in cin, line 135: aa770000
Error in cin, line 136: aam-4500
Error in cin, line 138: grupa9/zadanie2/aam-9100+aa090000+aa190000
Error in cin, line 140: grupa9/zadanie5/aam-4500+aam-8400
Error in cin, line 141: aa810000
Error in cin, line 142: grupa8/zadanie1/aam-0800+xzxx3700
Error in cin, line 143: aa970000
Error in cin, line 144: grupa9/zadanie4/aa090000+aam-8000+aa090000
Error in cin, line 145: grupa1/zadanie1/xzxx1700+aam-9300+xzxx1700
Error in cin, line 146: grupa9/zadanie1/aa780000+aa050000+aa050000
Error in cin, line 147: aa350000
Error in cin, line 148: grupa6/zadanie5/aa200000+xzxx2800+xzxx2800
Error in cin, line 149: grupa6/zadanie3/xzxx2100+xzxx2100
Error in cin, line 150: grupa6/zadanie1/xzxx8500+xzxx8500
Error in cin, line 151: aa440000
Error in cin, line 153: grupa3/zadanie4/xzxx3800+xzxx3800+xzxx3800
Error in cin, line 154: grupa6/zadanie2/aa090000+xzxx5200
Error in cin, line 157: grupa9/zadanie2/aa150000+xzxx9400+aa150000
Error in cin, line 158: aam-1100
Error in cin, line 159: grupa1/zadanie5/xzxx5200+aa650000
Error in cin, line 162: aam-8000
Error in cin, line 163: aa330000
Error in cin, line 164: grupa9/zadanie2/aa860000+aa860000
Error in cin, line 165: grupa3/zadanie5/xzxx3600+xzxx8500+xzxx8500
Error in cin, line 168: aa240000
Error in cin, line 171: aa780000
Error in cin, line 172: grupa5/zadanie4/xzxx9400+aa680000+aa160000
Error in cin, line 173: aa990000
Error in cin, line 174: grupa9/zadanie3/aa820000+aa150000+aa740000
Error in cin, line 175: grupa7/zadanie2/aa750000+xzxx2800+xzxx2800
Error in cin, line 176: aam-4500
Error in cin, line 177: grupa9/zadanie3/aa440000+aa440000
Error in cin, line 178: aa060000
Error in cin, line 179: aa030000
Error in cin, line 180: aa870000
Error in cin, line 181: grupa9/zadanie3/aa830000+aa830000+aa830000
Error in cin, line 182: grupa2/zadanie5/xzxx8500+xzxx8500+xzxx2800
Error in cin, line 183: grupa7/zadanie6/aa630000+xzxx7300+aa330000
Error in cin, line 184: grupa1/zadanie6/xzxx1400+aa410000+aa410000
Error in cin, line 185: grupa7/zadanie4/xzxx1400+xzxx1400
Error in cin, line 186: aa120000
Error in cin, line 187: aa950000
Error in cin, line 188: aa570000
Error in cin, line 190: grupa1/zadanie1/xzxx5300+xzxx5300+aa720000
Error in cin, line 191: aa580000
Error in cin, line 192: aa700000
Error in cin, line 193: aa750000
Error in cin, line 195: grupa5/zadanie1/aa020000+xzxx3800
Error in cin, line 196: grupa9/zadanie3/aa950000+aa640000
Error in cin, line 197: aa980000
Error in cin, line 198: aam-1100
Error in cin, line 198: aam-1100
Error in cin, line 199: aam-5500
Error in cin, line 200: aa410000
Error in cin, line 201: grupa1/zadanie5/xzxx6600+aa630000+aa630000
Error in cin, line 202: aa060000
Error in cin, line 203: aa060000
Error in cin, line 205: aa860000
Error in cin, line 206: aa740000
Error in cin, line 206: aa740000
Error in cin, line 207: grupa6/zadanie2/aam-1100+aam-1100+xzxx0000
Error in cin, line 209: grupa5/zadanie2/xzxx6000+aam-9300+aa220000
Error in cin, line 210: aam-9100
Error in cin, line 211: grupa3/zadanie6/xzxx8500+xzxx7300+xzxx7300
Error in cin, line 212: aa100000
Error in cin, line 212: aa100000
Error in cin, line 213: aam-1100
Error in cin, line 214: grupa9/zadanie3/xzxx3700+aa200000+aa830000
Error in cin, line 216: grupa7/zadanie6/aa630000+xzxx7300
Error in cin, line 218: grupa3/zadanie2/aam-5500 +aam-5500+aam-5500
Error in cin, line 219: grupa8/zadanie2/xzxx1400+xzxx1400+xzxx1400
Error in cin, line 222: aam-9100
Error in cin, line 223: grupa2/zadanie6/xzxx5200+xzxx5200+aa710000
Error in cin, line 224: aa100000
Error in cin, line 225: grupa5/zadanie6/xzxx6600+aa120000
Error in cin, line 226: grupa2/zadanie5/aa960000+xzxx6600
Error in cin, line 227: grupa5/zadanie3/xzxx4700+aa560000+aa560000
Error in cin, line 230: grupa9/zadanie2/aam-8000+aa350000+aam-8000
Error in cin, line 231: aa120000
Error in cin, line 232: aam-9000
Error in cin, line 233: grupa6/zadanie4/xzxx5300+aa860000
Error in cin, line 234: aa070000
Error in cin, line 235: grupa4/zadanie3/xzxx5300+xzxx2800+xzxx2800
Error in cin, line 236: grupa9/zadanie6/aa300000+aa300000
Error in cin, line 237: aa010000
Error in cin, line 238: grupa6/zadanie3/xzxx9400+xzxx2100+xzxx2100
Error in cin, line 239: grupa5/zadanie5/xzxx5300+xzxx5300+xzxx5300
Error in cin, line 241: grupa1/zadanie4/xzxx8500+aa540000
Error in cin, line 245: grupa3/zadanie4/xzxx3700+aa960000+xzxx3700
Error in cin, line 247: aa130000
Error in cin, line 249: aa160000
Error in cin, line 250: grupa2/zadanie3/aa490000+aa490000+xzxx5100
Error in cin, line 252: aa680000
Error in cin, line 253: grupa9/zadanie6/aa640000+aa640000+aa810000+aa640000
Error in cin, line 255: aam-9300
Error in cin, line 256: grupa7/zadanie4/xzxx8800+aam-1800
Error in cin, line 257: aa810000
Error in cin, line 259: aa580000
Error in cin, line 260: aa720000
Error in cin, line 261: aa820000
Error in cin, line 262: grupa9/zadanie5/aa650000+aa780000+aa780000
Error in cin, line 264: grupa2/zadanie1/xzxx4700+aa860000
Error in cin, line 265: aa670000
Error in cin, line 265: aa670000
Error in cin, line 266: aa760000
Error in cin, line 267: aa240000
Error in cin, line 268: aam-8400
Error in cin, line 269: grupa8/zadanie5/xzxx6000+xzxx6000+xzxx6000
Error in cin, line 270: aa410000
Error in cin, line 271: grupa8/zadanie3/xzxx8800+xzxx8800+xzxx8500
Error in cin, line 272: aa400000
Error in cin, line 273: grupa2/zadanie6/xzxx7300+aa760000
Error in cin, line 274: grupa9/zadanie2/aam-1100+aa420000
Error in cin, line 275: aa160000
Error in cin, line 276: aa060000
Error in cin, line 278: grupa9/zadanie1/aa570000+aam-8000
Error in cin, line 279: aam-8000
Error in cin, line 280: grupa4/zadanie6/xzxx5100+aa060000+aa060000
Error in cin, line 281: aa690000
Error in cin, line 283: grupa1/zadanie3/xzxx2800+aa070000+aa070000
Error in cin, line 284: aam-8000
Error in cin, line 285: aa440000
Error in cin, line 286: grupa6/zadanie5/xzxx2900+aa480000
Error in cin, line 287: aa390000
Error in cin, line 288: grupa3/zadanie1/aa070000+aa270000+aa270000+aa810000
Error in cin, line 291: aa190000
Error in cin, line 292: grupa3/zadanie6/xzxx2900+xzxx2900+aam-2600
Error in cin, line 294: grupa2/zadanie1/xzxx2900+xzxx2900
Error in cin, line 296: grupa2/zadanie5/xzxx6000+xzxx5100
Error in cin, line 297: aa460000
Error in cin, line 297: aa460000
Error in cin, line 298: grupa9/zadanie1/xzxx3600+aa540000+aa540000
Error in cin, line 299: grupa3/zadanie6/aa050000+xzxx6600
Error in cin, line 300: aa960000
Error in cin, line 301: aa400000
Error in cin, line 302: aa810000
Error in cin, line 303: aa030000
Error in cin, line 304: grupa2/zadanie1/aa240000+xzxx6600+aa700000
Error in cin, line 305: grupa2/zadanie1/aa410000+aa050000+aa410000+aa410000+aa100000
Error in cin, line 306: aam-5900
Error in cin, line 307: grupa7/zadanie1/aa090000+xzxx3600
Error in cin, line 309: aa350000
Error in cin, line 310: grupa1/zadanie3/aa980000+xzxx4700+aa790000
Error in cin, line 311: grupa6/zadanie2/aa690000+aa490000+aa490000+aa690000+aa690000
Error in cin, line 313: grupa7/zadanie3/xzxx4700+aa420000 +xzxx4700
Error in cin, line 316: grupa7/zadanie6/xzxx8500+aam-4500+xzxx4700
Error in cin, line 318: grupa5/zadanie4/xzxx5200+xzxx5200+xzxx5200
Error in cin, line 319: grupa6/zadanie4/aam-8000+xzxx4700
Error in cin, line 320: grupa9/zadanie2/xzxx9400+aam-4500+xzxx9400
Error in cin, line 322: grupa4/zadanie4/aa090000+aa420000+xzxx8500
Error in cin, line 324: grupa1/zadanie5/aa460000+xzxx5200
Error in cin, line 325: aa960000
Error in cin, line 326: aa350000
Error in cin, line 326: aa350000
Error in cin, line 327: aa710000
Error in cin, line 328: aa460000
Error in cin, line 329: grupa1/zadanie3/xzxx7300+xzxx7300+aa980000
Error in cin, line 330: aa790000
Error in cin, line 330: aa790000
Error in cin, line 331: grupa7/zadanie5/xzxx2900+xzxx2900+aa720000
Error in cin, line 333: aa310000
Error in cin, line 334: aa410000
Error in cin, line 335: grupa2/zadanie2/xzxx5200+aa490000
Error in cin, line 339: grupa5/zadanie3/xzxx3700+xzxx9400
Error in cin, line 341: aa830000
Error in cin, line 343: aa250000
Error in cin, line 344: aa440000
Error in cin, line 345: grupa9/zadanie3/aam-8400+aa060000
Error in cin, line 347: aa890000
Error in cin, line 348: aa040000
Error in cin, line 349: grupa7/zadanie4/xzxx0000+xzxx0000
Error in cin, line 350: grupa9/zadanie6/aa630000+aa630000+aa070000
Error in cin, line 351: aa150000
Error in cin, line 352: grupa1/zadanie3/xzxx5200+aa090000+xzxx5200
Error in cin, line 353: grupa7/zadanie4/xzxx9400+xzxx6000+xzxx6000
Error in cin, line 354: aa710000
Error in cin, line 355: grupa9/zadanie4/aa030000+aa030000+aa760000
Error in cin, line 357: grupa9/zadanie5/aa190000+aa190000+aa190000
Error in cin, line 358: grupa2/zadanie4/xzxx5200+aa920000
Error in cin, line 359: aa420000
Error in cin, line 360: aa990000
Error in cin, line 361: aa810000
Error in cin, line 362: grupa1/zadanie2/aa620000+aa620000+xzxx5200
Error in cin, line 363: grupa9/zadanie3/xzxx5200+aa950000+xzxx3700
Error in cin, line 364: grupa1/zadanie5/xzxx2900+aa480000+xzxx2900
Error in cin, line 365: aa630000
Error in cin, line 367: grupa9/zadanie5/aa090000+xzxx5300
Error in cin, line 368: grupa5/zadanie3/xzxx5200+aa920000+aa920000
Error in cin, line 370: aa090000
Error in cin, line 372: aa250000
Error in cin, line 372: aa250000
Error in cin, line 373: aa810000
Error in cin, line 375: grupa3/zadanie4/aa010000+aa010000+xzxx2800
Error in cin, line 377: grupa9/zadanie6/aa050000+aa050000
Error in cin, line 381: aa670000
Error in cin, line 383: grupa4/zadanie1/xzxx0000+aa030000+aa030000
Error in cin, line 384: grupa4/zadanie5/xzxx1700+xzxx1700+aa500000
Error in cin, line 385: aa240000
Error in cin, line 386: grupa4/zadanie2/xzxx3800+xzxx3800
Error in cin, line 387: grupa6/zadanie6/xzxx6600+xzxx6600
Error in cin, line 388: grupa8/zadanie4/xzxx5200+xzxx5200
Error in cin, line 389: grupa2/zadanie6/xzxx7300+xzxx7300
Error in cin, line 390: grupa6/zadanie4/xzxx5200+aa490000+xzxx5200
Error in cin, line 391: grupa2/zadanie1/xzxx3800+xzxx3800+aa120000
Error in cin, line 393: grupa9/zadanie2/aa570000+aa570000
Error in cin, line 394: aa750000
Error in cin, line 396: aa560000
Error in cin, line 397: grupa3/zadanie3/xzxx3800+aa490000
Error in cin, line 398: grupa1/zadanie6/xzxx3700+aa070000+aa250000
Error in cin, line 399: grupa7/zadanie5/xzxx3600+aa330000+aa480000
Error in cin, line 401: aa350000
Error in cin, line 402: aa100000
Error in cin, line 402: aa100000
Error in cin, line 403: grupa2/zadanie5/aa420000+xzxx8500+aam-1100
Error in cin, line 404: grupa9/zadanie3/aa770000+aam-9000+aa740000
Error in cin, line 405: aa390000
Error in cin, line 406: aam-9000
Error in cin, line 406: aam-9000
Error in cin, line 407: aam-2600
Error in cin, line 407: aam-2600
Error in cin, line 408: aam-9000
Error in cin, line 411: grupa2/zadanie2/aa320000+aam-0800+xzxx4700
Error in cin, line 412: grupa6/zadanie6/aa790000+aa790000+xzxx5100
Error in cin, line 413: aa740000
Error in cin, line 414: aa480000
Error in cin, line 415: grupa6/zadanie2/xzxx7300+aam-4300
Error in cin, line 417: aam-9100
Error in cin, line 418: grupa9/zadanie2/aa100000+aa100000+xzxx9400
Error in cin, line 419: aa610000
Error in cin, line 420: grupa5/zadanie5/xzxx1700+xzxx1700
Error in cin, line 421: grupa3/zadanie5/aa320000+aa440000+xzxx3700
Error in cin, line 422: grupa8/zadanie5/aa060000+xzxx9400+xzxx5200
Error in cin, line 424: aa610000
Error in cin, line 425: aa270000
Error in cin, line 425: aa270000
Error in cin, line 426: grupa7/zadanie3/aa410000+xzxx6000+xzxx6000
Error in cin, line 427: aa580000
Error in cin, line 428: aa560000
Error in cin, line 430: aa250000
Error in cin, line 431: aa300000
Error in cin, line 432: aa070000
Error in cin, line 433: aa350000
Error in cin, line 434: grupa1/zadanie2/xzxx3800+xzxx3800
Error in cin, line 435: grupa8/zadanie3/xzxx3700+aam-1100+aam-1100
Error in cin, line 437: aa190000
Error in cin, line 438: aa340000
Error in cin, line 439: aa830000
Error in cin, line 440: grupa2/zadanie2/aa480000+xzxx9400+xzxx9400
Error in cin, line 442: aa020000
Error in cin, line 443: grupa4/zadanie3/xzxx2800+xzxx2800+aa340000
Error in cin, line 445: grupa2/zadanie3/aa830000+aa830000+xzxx5200
Error in cin, line 447: grupa9/zadanie6/aa790000+aa090000+aa090000
Error in cin, line 448: grupa9/zadanie5/aa960000+aa350000
Error in cin, line 450: grupa9/zadanie4/aa100000+aa100000+xzxx2800
Error in cin, line 451: aa780000
Error in cin, line 452: grupa9/zadanie1/aa070000+xzxx5200
Error in cin, line 453: grupa9/zadanie5/aa690000+aa690000+aa690000
Error in cin, line 454: grupa7/zadanie4/xzxx0000+aa970000
Error in cin, line 455: grupa2/zadanie5/aa980000+xzxx9400
Error in cin, line 457: aa760000
Error in cin, line 458: grupa2/zadanie1/aa350000+xzxx8800+aa200000
Error in cin, line 462: grupa9/zadanie3/xzxx2100+xzxx2100+xzxx3800
Error in cin, line 463: grupa3/zadanie4/xzxx2100+xzxx2100
Error in cin, line 464: grupa9/zadanie6/xzxx9400+xzxx9400+xzxx9400
Error in cin, line 465: aa010000
Error in cin, line 466: aa890000
Error in cin, line 467: grupa5/zadanie4/aa050000+xzxx1700
Error in cin, line 468: aa330000
Error in cin, line 470: grupa3/zadanie4/aa460000+xzxx9400
Error in cin, line 471: aa870000
Error in cin, line 472: aam-0800
Error in cin, line 473: aam-5900
Error in cin, line 474: grupa6/zadanie2/aa460000+xzxx3700+aa460000
Error in cin, line 475: grupa9/zadanie2/aa760000+xzxx1400+aa120000
Error in cin, line 476: grupa8/zadanie5/aa870000+xzxx3600+aam-4300
Error in cin, line 477: aam-5500
Error in cin, line 478: aa710000
Error in cin, line 478: aa710000
Error in cin, line 479: aa030000
Error in cin, line 480: grupa3/zadanie2/aa780000+xzxx1700
Error in cin, line 481: grupa5/zadanie1/xzxx2800+xzxx2800
Error in cin, line 482: grupa9/zadanie1/aam-8000+aa420000
Error in cin, line 484: grupa6/zadanie3/xzxx2100+aam-9300+xzxx2100
Error in cin, line 485: grupa5/zadanie2/xzxx4700+xzxx3600+xzxx4700
Error in cin, line 486: aa810000
Error in cin, line 488: aa020000
Error in cin, line 489: aa610000
Error in cin, line 491: grupa3/zadanie6/aa680000+xzxx3600+aa490000
Error in cin, line 492: grupa6/zadanie5/aa230000+xzxx6000
Error in cin, line 493: aa420000
Error in cin, line 493: aa420000
Error in cin, line 495: aa500000
Error in cin, line 498: aa160000
Error in cin, line 500: aa040000
Error in cin, line 500: aa040000
