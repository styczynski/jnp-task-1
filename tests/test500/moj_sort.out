060000;1;
220000;1;
230000;1;
340000;1;
670000;1;
690000;1;
710000;1;
740000;1;
770000;1;
780000;1;
830000;1;
890000;1;
920000;2;
990000;1;
m-9300;1;
